notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -117,7 +117,8 @@
    "metadata": {},
```

Comment:
<p>how we *can accurately* discriminate <em>between</em> these 4 hand *positions.*</p><p> </p><p>The baseline accuracy is defined as the accuracy we get by randomly guessing the signs. In our case, the baseline accuracy is 25%.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>